core/domain/exp_domain.py

```diff
@@ -207,6 +210,24 @@ def _validate_customization_args_and_values(
             pass
 
 
+def retrieve_old_state_name(state_name, renamed_cmds):
```

Comment:
I removed the function since we aren't duplicating anymore. We are doing the computation here and passing the mapping around. I'll include tests in the next commit.